### DISCUSSION TITLE
Graph

### DISCUSSION THREAD

**Discussion by ankit-akash** [ankit-akash/DSA#14] [Q&A] [answered]
A complete graph can have ___________spanning trees?

**Comment by Preetiraj3697** [comment]
nn-2 number
From a complete graph, by removing maximum e - n + 1 edges, we can construct a spanning tree. A complete graph can have maximum nn-2 number of spanning trees.

**Comment by alperencubuk** [comment, marked as answer]
n^(n-2)
